pkg/node/address.go

```diff
@@ -46,6 +46,7 @@ var (
 	ipv6NodePortAddrs map[string]net.IP // iface name => ip addr
 	ipv4AllocRange    *cidr.CIDR
 	ipv6AllocRange    *cidr.CIDR
+	externalAddress   net.IP
```

Comment:
Could you add a comment saying that this is k8s Node ExternalIP?

Reply:
Done.